[CN-360] Add info for multiple pod and service label support (#534)
hazelcast/hz-docs: docs/modules/deploy/pages/configuring-kubernetes.adoc

* `namespace`: Kubernetes Namespace where Hazelcast is running; if not specified, the value is taken from the environment variables `KUBERNETES_NAMESPACE` or `OPENSHIFT_BUILD_NAMESPACE`. If those are not set, the namespace of the Pod will be used (retrieved from `/var/run/secrets/kubernetes.io/serviceaccount/namespace`).
* `service-name`: service name used to scan only Pods connected to the given service; if not specified, then all Pods in the namespace are checked
* `service-label-name`, `service-label-value`: service label and value used to tag services that should form the Hazelcast cluster together. Can support comma separated multiple values, an example is "label-1,label-2". Note that the number of elements in` service-label-name` and `service-label-value` must be equal. 

> * `pod-label-name`, `pod-label-value`: pod label and value used to tag pods that should form the Hazelcast cluster together. Can support comma separated multiple values, an example is "label-1,label-2". Note that the number of elements in` pod-label-name` and `pod-label-value` must be equal. 

Reviewer: ```suggestion
* `pod-label-name`, `pod-label-value`: pod label and value used to tag Pods, which together form the Hazelcast cluster. These properties can support multiple comma-separated values. For example: "label-1,label-2". You must use the same number of elements in `pod-label-name` as `pod-label-value`. 
```
